src/components/Home.js

```diff
@@ -7,23 +7,18 @@ const Home = () => {
   const fetchData = async () => {
     const response = await axios.get("http://localhost:3000/messages");
     dispatch(setGreetings(response.data));
-    
   };
   useEffect(() => {
     fetchData();
   },[]);
-  const greeting = useSelector(state => state.greeting.greeting);
-    
-    return (
-        <div>
+  const greeting = useSelector(state => state.greeting.greeting.message);
+  console.log(greeting);
```

Comment:
- [Optional] Kindly remove all console.log from your file. It is best practice not to commit console.log into your repo.